spec/features/user_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe 'User', type: :feature do
+  before :all do
+    @user = User.create(name: 'new', email: 'new@gmail.com', password: '123123')
+  end
+
+  describe 'Log in page' do
+    it 'should log in with credentials' do
```

Comment:
Required

Please test the other features of the app as well, such as creating categories and expenses.